go-controller/pkg/ovn/egressfirewall_test.go

```diff
@@ -187,10 +188,17 @@ var _ = ginkgo.Describe("OVN EgressFirewall Operations", func() {
 									ObjectMeta: newObjectMeta(node1Name, ""),
 								},
 							},
+						},
+						&v1.NamespaceList{
```

Comment:
instead of adding the namespace and everything that goes with it could we just add the namespaces addressSet to the db?